### ISSUE TITLE
Bug: HackforLABot moving issues to wrong columns

### ISSUE THREAD

**Issue by SAUMILDHANKAR** [hackforla/website#2956]
### Overview
As a developer, I want to fix the bug where the HackforLA bot is moving the issues to wrong columns so as to ensure that the bot actions are error free and helpful.

### Action Items
- [x] Review how and when HackforLA bot moves issues between different columns
- [x] Analyze why the issue was moved from Done to QA in #2560 and #2707 and why the issue was moved from New Issue Approval to Done in #2400
- [x] Review if changes to the following files might help in fixing the bug: [Move closed issues yaml file](https://github.com/hackforla/website/blob/gh-pages/.github/workflows/move-closed-issues.yaml),
[Sost closed issues js file](https://github.com/hackforla/website/blob/gh-pages/github-actions/move-closed-issues/sort-closed-issues.js)
- [x] Analyze if the project board automations might interfere with GitHub actions
- [x] Check if there are any other scenarios where the issue might be moved unexpectedly
- [x] Test the changes in your own repo and ensure existing functionalities are not impacted
- [x] Make your recommendations or proposed solutions to tech leads or in a team meeting
- [x] Create a PR that fixes this bug

### Resources/Instructions
Issues where bug was encountered: 
    - Issues were moved from Done to QA: [Case 1](https://github.com/hackforla/website/issues/2560), [Case 2](https://github.com/hackforla/website/issues/2707)
    - [Issue was moved from New Issue Approval to Done](https://github.com/hackforla/website/issues/2400)]
[Helpful issue for review](https://github.com/hackforla/website/issues/2206)
[Move closed issues yaml file](https://github.com/hackforla/website/blob/gh-pages/.github/workflows/move-closed-issues.yaml)
[Sost closed issues js file](https://github.com/hackforla/website/blob/gh-pages/github-actions/move-closed-issues/sort-closed-issues.js)

**Comment by Zak234** [comment]
Availability: All day
ETA: 2-3 days

**Comment by Zak234** [comment]
I looked at the issues and it seems like the reason issues #2560 and #2707 were moved was due to poor labelling on the issues. For #2400, it looks like it was just an issue with the person accidentally closing and reopening their issue which caused the action to run. I discussed with the tech leads and Bonnie about whether we should change which labels send and prevent issues from going to the QA column and it was decided that the current code suits our needs.
